### DISCUSSION TITLE
Commercial use

### DISCUSSION THREAD

**Discussion by bertoluccimauricio** [trakt/trakt-api#455] [Q&A] [answered]
I'm sorry, I'm sure this information is easy somewhere, but I couldn't find it.
Basic question, can the API data be used commercially, that is, can I build an APP and charge for it?
Thanks

**Comment by rectifyer** [comment, marked as answer]
Yes, that is fine. Check out https://trakt.tv/branding and we only ask that you implement either Trakt sign in and check ins or scrobbling depending how your app works.
